src/sentry/models/organizationmembermapping.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import annotations
+
+from django.conf import settings
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import FlexibleForeignKey, Model, sane_repr
+from sentry.db.models.base import control_silo_only_model
+from sentry.db.models.fields.hybrid_cloud_foreign_key import HybridCloudForeignKey
+from sentry.models.organizationmember import InviteStatus
+from sentry.roles import organization_roles
+
+
+@control_silo_only_model
+class OrganizationMemberMapping(Model):
+    """
+    This model resides exclusively in the control silo, and will
+    - map a user or an email to a specific organization to indicate an organization membership
+    """
+
+    __include_in_export__ = True
```

Comment:
```suggestion
    __include_in_export__ = False
```

I'm not sure this will work in an export, but then again all of import/export has to change for multi-region.